XmlFormatter/src/Windows/VersionInformation.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Diagnostics;
+using System.Windows.Forms;
+using XmlFormatter.src.Manager;
+
+namespace XmlFormatter.src.Windows
+{
+    /// <summary>
+    /// Version information windows
+    /// </summary>
+    public partial class VersionInformation : Form
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5495773)